src/screens/LinksScreen.tsx

```diff
@@ -3,12 +3,23 @@ import { StyleSheet } from 'react-native';
 import { ScrollView } from 'react-native-gesture-handler';
 import OptionButton from '../components/OptionButton';
 
-export default function LinksScreen() {
+export default function LinksScreen({
```

Comment:
I'm not sure if this optimisation is needed - this was more of a placeholder component and as we have started to add more specific screens, they will not really be able to re-use this. Instead, you can just create a `ConfigurationScreen` instead and just add some placeholder for that, since it will be custom design. If we see later down the line that this is needed, we can use it.